tests/testthat/test-caching.R

```diff
@@ -1,11 +1,7 @@
-globalassign <- function(...) {
-  for (x in c(...)) assign(x, eval.parent(parse(text = x)), .GlobalEnv)
-}
-
 test_that("Caching works", {
   calcCacheExample <- function() return(list(x = as.magpie(1), description = "-", unit = "-"))
   globalassign("calcCacheExample")
-  localConfig(globalenv = TRUE, ignorecache = FALSE, .verbose = FALSE)
```

Comment:
why is `globalenv = TRUE ` not required anymore? isn`t `calcCacheExample` still assigned to it?

Reply:
setup.R sets `globalenv = TRUE` for all tests.